issue found on: analyser/aggregator.go:111:2: rangeValCopy

an issue has been detected in source code: ./analyser/aggregator.go:111:2: rangeValCopy: each iteration copies 144 bytes (consider pointers or indexing)

--- a/analyser/aggregator.go
+++ b/analyser/aggregator.go
@@ -113,9 +113,9 @@ func readAggregatorLogFile(filename string) ([]AggregatorLogEntry, error) {
 func filterConsumedMessages(entries []AggregatorLogEntry) []AggregatorLogEntry {
 	consumed := []AggregatorLogEntry{}
 
-	for _, entry := range entries {
-		if entry.Message == consumedFilter && entry.Group != "" {
-			consumed = append(consumed, entry)
+	for i := range entries {
+		if entries[i].Message == consumedFilter && entries[i].Group != "" {
+			consumed = append(consumed, entries[i])
 		}
 	}
 	return consumed

